src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
First of all, you should add a `private` access modifier, don't use the default one here. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#pay-attention-to-access-modifiers).

Then, pay attention to the naming. You should name the fields 'color' and 'number' correspondingly, without the unnecessary prefix 'ball', because the fields are already encapsulated within the entity 'Ball'. And it was explicitly mentioned in the requirements.

Thus, the final code:
```suggestion
   private String color;
   private int number;
```